### PR TITLE
Fix test error response.

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
Ready

## Related Issues
fixes: the build...

## Description
Send JSON object while grabbing playbook error.

### CODE REVIEW

**Comment by amshamah419** [comment]
Some of @guykeller's commits will show up here. I needed to use his branch to replicate the problem. I reverted changes to master so his PR is not affected.

File: Integrations/SplunkPy/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+-
```
Comment (yaakovi) [inline Integrations/SplunkPy/CHANGELOG.md]: revert!

File: Integrations/SplunkPy/CHANGELOG.md [hunk 2]
```diff
@@ -21,4 +21,4 @@ Added the *Earliest time to fetch* and *Latest time to fetch* parameters, which
 -
 
 ## [19.9.0] - 2019-09-04
-- Added the *Fetch limit* parameter to the instance configuration, which specified the maximum number of results to fetch.
+- Added the *Fetch limit* parameter to the instance configuration, which specified the maximum number of results to fetch.
```
Comment (amshamah419) [inline Integrations/SplunkPy/CHANGELOG.md]: I don't even know why this is different...

Comment (yaakovi) [reply Integrations/SplunkPy/CHANGELOG.md]: master has empty line at the end (don't know why it doesn't show here...)

**Pull request merged**